test/common/router/scoped_rds_test.cc

```diff
@@ -323,7 +339,10 @@ route_configuration_name: foo_routes
   EXPECT_EQ(1UL,
             server_factory_context_.scope_.counter("foo.scoped_rds.foo_scoped_routes.config_reload")
                 .value());
-  EXPECT_EQ(getScopedRouteMap().size(), 2);
+  EXPECT_EQ(2UL, server_factory_context_.scope_
+                     .gauge("foo.scoped_rds.foo_scoped_routes.all_scopes",
```

Comment:
You can snapshot this gauge in the base class, to make the boilerplate each time you access it less verbose.

Reply:
Yes, that makes sense.